metamorph/plugins/morph_resultsdb.py

```diff
@@ -0,0 +1,225 @@
+#!/usr/bin/python
+import argparse
+import logging
+import logging.config
+import json
+import time
+import os
+import requests
+
+from metamorph.lib.logging_conf import setup_logging
+
+
+class ResultsDBApiException(Exception):
+    pass
+
+
+class ResultsDBApi(object):
+    """
+    Class to communicate and process data with resultsDB
+    """
+    TIMEOUT_LIMIT = 7200  # Wait 2 hours maximally
+    RESULTSDB_LIMITER = 10
+    MINUTE = 60  # 60 seconds
+
+    def __init__(self, job_names, component_nvr, test_tier, resultsdb_api_url):
+        self.resultsdb_api_url = resultsdb_api_url
+        self.job_names = job_names
+        self.job_names_result = {}
+        self.tier_tag = True
+        self.url_options = {'CI_tier': test_tier, 'item': component_nvr}
+
+    def get_test_tier_metadata(self):
```

Comment:
Good catch. Yes I want to use it in the main function.
I'll think about a new name.